tools/unicode/py/preparseucd.py

```diff
@@ -624,6 +624,17 @@ def ParsePropertyAliases(in_file):
   AddEnumeratedValue(prop, "Limited_Use")
   AddEnumeratedValue(prop, "Inclusion")
   AddEnumeratedValue(prop, "Recommended")
+  # Indic_Conjunct Break. See UAX #29 and
+  # https://www.unicode.org/reports/tr44/tr44-33.html#Indic_Conjunct_Break
+  name = "InCB"
+  prop = ("Enumerated", ["InCB", "Indic_Conjunct_Break"], set(), {})
+  _properties[name] = prop
+  _properties[NormPropName(name)] = prop
+  SetDefaultValue(name, "None")
+  AddEnumeratedValue(prop, "None")
+  AddEnumeratedValue(prop, "Consonant")
+  AddEnumeratedValue(prop, "Extend")
+  AddEnumeratedValue(prop, "Linker")
```

Comment:
Please try to remove these lines and see if the script still works. This property is defined in the regular UCD files (PropertyAliases.txt & PropertyValueAliases.txt), so they should be parsed just fine by this function and the following function.

We should only need to manually define a property here if it is ICU-specific (gcm, lccc, ...) or if it's outside of the UCD proper (like Identifier_Type from UTS 39).

Reply:
Tried, and we don't need these lines :-)